packages/amplify-graphql-conversation-transformer/src/__tests__/__snapshots__/amplify-graphql-conversation-transformer.test.ts.snap

```diff
@@ -1,370 +1,5 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`ConversationTransformer parameterized tests should transform a conversation route with inference configuration 1`] = `
```

Comment:
The test cases generating these snapshots were/are already covered by other test cases.